docs/concepts/resolution.md

```diff
@@ -298,9 +298,9 @@ as if they do not exist.
 
 ## Source distribution
 
-Most packages publish their source distributions as gzip tarball (`.tar.gz`) or zip (`.zip`)
-archives. While less common, other formats are also supported when reading and extracting source
-distributions:
+[PEP 625](https://peps.python.org/pep-0625/) specifies that packages must distribute source
+distributions as gzip tarball (`.tar.gz`) archives. As before this PEP, other formats were also
+allowed, uv supports reading and extracting archives that use the following formats:
```

Comment:
```suggestion
[PEP 625](https://peps.python.org/pep-0625/) specifies that packages must distribute source
distributions as gzip tarball (`.tar.gz`) archives. Prior to this specification, other archive formats
were allowed so uv also supports reading and extracting archives in the following formats:
```

Reply:
Maybe something like this?

Reply:
Woops sorry, I force-pushed before your review, since it was still in draft (damn you're fast! 😄). Will incorporate your suggestions though, thanks!